File: problems/1/solution.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        var d = new Dictionary<int, int>();
        
        for (var i = 0; i < nums.Length; i++) {
            var compl = target - nums[i];
            
            if (d.ContainsKey(compl)) {
                return new int[] {d[compl], i};
            }
            
            d[nums[i]] = i;
        }
        
        return new int[]{};
    }
}

File: problems/1/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hm = {}
        
        for i in range(len(nums)):
            compl = target - nums[i]
            
            if compl in hm:
                return [hm[compl], i]
            
            hm[nums[i]] = i